tests/Stormpath/Tests/Resource/AccountCreationPolicyTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Stormpath\Tests\Resource;
+
+
+use Stormpath\Stormpath;
+
+class AccountCreationPolicyTest extends \Stormpath\Tests\BaseTest {
+
+    private static $directory;
+
+    private static $acp;
+
+    public static function setUpBeforeClass()
+    {
+        parent::setUpBeforeClass();
+        self::$directory = \Stormpath\Resource\Directory::create(array('name' => 'Another random directory' .md5(time().microtime().uniqid())));
+        self::$acp = self::$directory->accountCreationPolicy;
+    }
+
+    private function enableAndTest($property)
+    {
+        self::$acp->$property = STORMPATH::ENABLED;
+        self::$acp->save();
+
+        $acp = self::$directory->accountCreationPolicy;
+        $this->assertEquals(STORMPATH::ENABLED, $acp->$property);
+    }
+
+    private function disableAndTest($property)
+    {
+        self::$acp->$property = STORMPATH::DISABLED;
+        self::$acp->save();
+
+        $acp = self::$directory->accountCreationPolicy;
+        $this->assertEquals(STORMPATH::DISABLED, $acp->$property);
+    }
+
+    /**
+     * @test
+     */
+    public function it_should_allow_changing_verification_email_status()
+    {
+        $this->assertEquals(STORMPATH::DISABLED, self::$acp->verificationEmailStatus);
+
+
+        $this->enableAndTest('verificationEmailStatus');
+        $this->disableAndTest('verificationEmailStatus');
```

Comment:
No need to change this, but if you only had one function called `changeStatusAndTest` where you also pass the status as an argument you could have used that instead of duplicating code (general good practice to have in mind, but not required to change here).